Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/CommentDAO.java

```diff
@@ -54,6 +55,18 @@ public List<Comment> getAllByProcess(Process process) {
                 Collections.singletonMap("processId", process.getId()));
     }
 
+    /**
+     * Get all comments by task ordered by id ascending.
+     *
+     * @param task
+     *         The current task to get the comments for
+     * @return List of comments
+     */
+    public List<Comment> getAllByTask(Task task) {
+        return getByQuery("FROM Comment WHERE currentTask_id = :taskId ORDER BY id ASC",
```

Comment:
On Linux, table names are case sensitive and we decided that they should be all lowercase. So I assume, this must be `FROM comment` instead.

Reply:
I changed this but we should change also the other tables names, cause table name of every comment query starts with the upper case.

Reply:
Very surprising. Either there is a graceful handling in Hibernate, or nobody ever noticed it doesn’t work?

Reply:
There are even totally wrong table names here. It’s a chance that this is dead code. I create a separate pull request for that.

Reply:
This query is a HQL (Hibernate Query Language) and not a SQL query and an HQL is not case-sensitive or get formatted into the right and correct database query language. But if you create a raw query (not here but on other places) or writing a flyway migration you are writing an SQL query which is case-sensitive.

Reply:
> There are even totally wrong table names here. It’s a chance that this is dead code. I create a separate pull request for that.

Do not mix HQL with SQL queries! You can and maybe even must do things in HQL in an other way then in SQL! You can of course write everything in SQL but then you lost the independency from the underlaying database.

Reply:
https://github.com/kitodo/kitodo-production/pull/5692

Reply:
So i think it depends on your character configuration of database. If it ends with `_ci` so it will be case insensive. The [installation documentation](https://github.com/kitodo/kitodo-production/wiki/Installationsanleitung-f%C3%BCr-Kitodo.Production-3.4) has not such a specification. IMO the default is mysql specific and default is with `_ci` https://dev.mysql.com/doc/refman/8.0/en/charset-server.html.

Reply:
@matthias-ronge and @markusweigelt : I can only repeat: this query strings are written in HQL and not SQL, and so the queries itself are not relaying on the real used database and how the database is configured.

Reply:
Then my suggestion for change was wrong. I didn’t knew about HQL at all.